onebusaway-android/src/main/java/org/onebusaway/android/ui/SituationDialogFragment.java

```diff
@@ -154,16 +155,18 @@ public void onClick(DialogInterface dialog, int which) {
                     }
                 });
 
-        final androidx.appcompat.app.AlertDialog dialog = builder.create();
```

Comment:
what's the impact of using the built-in AlertDialog instead of the `androidx` version on an older Android device? (e.g. Android 5)

Reply:
it's actually android x, i have just imported it in imports line 30

`import androidx.appcompat.app.AlertDialog;`

instead of declaring an object with package name.

Reply:
Yes, i have tested it with non html tags strings & also for hyberlinks and it works fine.
![Screenshot from 2024-02-25 02-37-59](https://github.com/OneBusAway/onebusaway-android/assets/29693819/a1948dee-fb33-4840-b5b9-e295df42c7dd)